### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by yrunts** [Semantics3/semantics3-python#14]
Version 0.3.2 is not compatible with python 3, because of urlnorm library.

```
[GCC 4.8.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from semantics3 import Product
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/vagrant/.env/lib/python3.4/site-packages/semantics3/__init__.py", line 4, in <module>
    from .semantics3 import Semantics3Request
  File "/vagrant/.env/lib/python3.4/site-packages/semantics3/semantics3.py", line 3, in <module>
    from urlnorm import norm
  File "/vagrant/.env/lib/python3.4/site-packages/urlnorm.py", line 184
    MAX_IP=0xffffffffL
                     ^
SyntaxError: invalid syntax
```

**Comment by ryanio** [comment]
+1

**Comment by abishekk92** [comment]
@yrunts Thanks for reporting this. We're looking at how we can support Python 3 in the next release.

**Comment by trentniemeyer** [comment]
+1

**Comment by trentniemeyer** [comment]
I need to submit a pull request for this: but fixing this issue was a simple removal of an unnecessary library.  I have a fork of this repo here: https://github.com/BrandCards/semantics3-python.  (Also in there is a simple skus object for sku API access.

**Comment by billyp1980** [comment]
This will probably preclude me from using semantic3 at all, unfortunately.

**Comment by abishekk92** [comment]
This has been fixed in the latest release:[v0.3.5 ](https://pypi.python.org/pypi/semantics3). Apologies for the late fix, hope this helps :)
